Dockerfile

```diff
@@ -11,7 +11,13 @@ RUN apt-get update \
     && pip install ${PIP_OPTIONS} -U setuptools
 
 WORKDIR /workspaces
-COPY . .
+
+# Create (dummy) files required to run setup.py
+RUN mkdir optuna \
+    && touch README.md \
+    && echo '__version__ = "0.0.0"' > optuna/version.py
+
+COPY setup.py .
```

Comment:
You can sonsider these files as a dummy optuna.

Reply:
This dummy optuna is installed in the *editable* mode so we can overwrite it later.